### PR TITLE
Pull in remote scorer

### PR DESCRIPTION
**Pull request by TonyGiorgio**
One of these days we will do a proper builder, today is not one of those days.

### CODE REVIEW

File: mutiny-core/src/gossip.rs [hunk 1]
```diff
@@ -64,40 +83,82 @@ async fn get_gossip_data(
...
+        scorer,
     };
 
     Ok(Some(gossip))
 }
 
+/// Scorer is the scorer that gets pulled remotely
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Scorer {
+    pub value: Vec<u8>,
```
Comment (benthecarman) [inline mutiny-core/src/gossip.rs:139]: Would be better if we encode as hex or even better base64 to reduce bandwidth.

Comment (TonyGiorgio) [reply mutiny-core/src/gossip.rs:139]: Encoding will increase bandwidth

Comment (benthecarman) [reply mutiny-core/src/gossip.rs:139]: It's written as [1, 2, 3,..] in the json so hex is more effienct

Comment (TonyGiorgio) [reply mutiny-core/src/gossip.rs:139]: Damn you're right actually. I guess I can get rid of all the serialization completely then and just deal with raw bytes, no object?

Comment (benthecarman) [reply mutiny-core/src/gossip.rs:139]: Yeah that'd be best I guess

Comment (TonyGiorgio) [reply mutiny-core/src/gossip.rs]: I spent far too long messing with stupid byte responses that wouldn't properly decode, kept the json but swapped for base64.

**Review by benthecarman — APPROVED** [review]

**Pull request merged**